[jit] split remaining "easy" tests

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29249 [jit] split remaining "easy" tests**

This splits out all the tests that are "easy", leaving `TestJit`,
`TestScript`, the autogenerated tests, and a small docs test.

Splitting those into reasonable chunks is more effort which is less
mechanical.

Pull Request resolved: https://github.com/pytorch/pytorch/pull/29249

Differential Revision: [D18339007](https://our.internmc.facebook.com/intern/diff/D18339007)

suo committed:
[jit] split TestDataParallel, TestAsync

kostmo commented:
## CircleCI build failures summary
As of commit 295cc7d:
* 0/1 recognized as flaky
* 1/1 failures introduced in this PR



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=295cc7d5b08e4a6a561ef560d78e8ce402d91be6).

---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

suo committed:
Update on "[jit] split TestDataParallel, TestAsync"


Differential Revision: [D18339007](https://our.internmc.facebook.com/intern/diff/D18339007)

suo committed:
Update on "[jit] split remaining "easy" tests"


This splits out all the tests that are "easy", leaving `TestJit`,
`TestScript`, the autogenerated tests, and a small docs test.

Splitting those into reasonable chunks is more effort which is less
mechanical.

Pull Request resolved: https://github.com/pytorch/pytorch/pull/29249

Differential Revision: [D18339007](https://our.internmc.facebook.com/intern/diff/D18339007)

suo committed:
Update on "[jit] split remaining "easy" tests"


This splits out all the tests that are "easy", leaving `TestJit`,
`TestScript`, the autogenerated tests, and a small docs test.

Splitting those into reasonable chunks is more effort which is less
mechanical.

Pull Request resolved: https://github.com/pytorch/pytorch/pull/29249

Differential Revision: [D18339007](https://our.internmc.facebook.com/intern/diff/D18339007)

jamesr66a reviewed (approved):
don't blow anything up

suo committed:
Update on "[jit] split remaining "easy" tests"


This splits out all the tests that are "easy", leaving `TestJit`,
`TestScript`, the autogenerated tests, and a small docs test.

Splitting those into reasonable chunks is more effort which is less
mechanical.

Pull Request resolved: https://github.com/pytorch/pytorch/pull/29249

Differential Revision: [D18339007](https://our.internmc.facebook.com/intern/diff/D18339007)
